<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLicenseDialog</class>
 <widget class="QDialog" name="AddLicenseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление лицензии</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/images/favicon.ico</normaloff>:/images/favicon.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="DialogBtnBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="SerialLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>124</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Серийный(-е) номер(-а):</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="SerialTextEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="TypeLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="TypeLabel">
      <property name="text">
       <string>Тип:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="TypeComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="VendorLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="VendorLabel">
      <property name="text">
       <string>Производитель:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="VendorComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="SoftLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="SoftLabel">
      <property name="text">
       <string>Программный продукт:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="SoftComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>332</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="DurationLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="DurationLabel">
      <property name="text">
       <string>Срок действия лицензии (месяцев):</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="DurationSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>32767</number>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="UnlimitedCheckBox">
      <property name="text">
       <string>Бессрочная</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>132</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="CountLayout">
    <item>
     <widget class="QLabel" name="CountLabel">
      <property name="text">
       <string>Количество ПК:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="CountSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>DialogBtnBox</zorder>
  <zorder>SerialLabel</zorder>
  <zorder>SerialTextEdit</zorder>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>AddLicenseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>AddLicenseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
